new/INS1007Taqot_Maryam.xml

```diff
@@ -0,0 +1,99 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="INS1007Taqot_Maryam" xml:lang="en" type="ins">
```

Comment:
what is done is done, but we should avoid hyphens and underscores in xml:id and filenames, as is written in the Guidelines

Reply:
It seems that hyphen was generated by the system, could it be? Did I had to delete it manually?

Reply:
You must have entered Taqot Maryam (with a space) in the form, we cannot use spaces, these become the underscore, but we should not use underscores either. You should have entered TaqotMaryam as one word then there would be no error

Reply:
Ok, probably it is my mistake, then.

Reply:
I have added a prompt to the form